modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/data/provider/AnalyticsReportsDataProvider.java

```diff
@@ -90,24 +89,26 @@ public Map<String, AcquisitionChannel> getAcquisitionChannels(
 
 			Collection<Long> values = acquisitionChannels.values();
 
-			Stream<Long> valuesStream = values.stream();
+			Double total = Double.valueOf(0);
 
-			Double total = Double.valueOf(valuesStream.reduce(0L, Long::sum));
+			for (Long value : values) {
+				total += value;
+			}
 
-			Set<Map.Entry<String, Long>> entries =
-				acquisitionChannels.entrySet();
+			Map<String, AcquisitionChannel> acquisitionChannelMap =
```

Comment:
@slnn I think the original `Map<String, Long> acquisitionChannels` is not a good name. I'd call it either `acquisitionChannelValues` or `trafficAmounts`. For the latter one, it's because the map contains pairs of "acquisition channel name" mapping to "acquisition channel traffic amount".

For `trafficAmount`, see the second parameter of `AcquisitionChannel` constructor.